test/src/receiver/main.cpp

```diff
@@ -39,91 +39,127 @@ struct outp : public cadmium::out_port<Message_t>{};
-
-using logger_top=cadmium::logger::multilogger<log_messages, global_time>;
-
-
-/*******************************************/
-
-
-
-/********************************************/
-/****** APPLICATION GENERATOR *******************/
-/********************************************/
-string input_data_control = "../test/data/receiver/receiver_input_test.txt";
-const char * i_input_data_control = input_data_control.c_str();
-
-std::shared_ptr<cadmium::dynamic::modeling::model> generator = cadmium::dynamic::translate::make_dynamic_atomic_model<ApplicationGen, TIME, const char* >("generator" , std::move(i_input_data_control));
-
-/********************************************/
-/****** RECIEVER *******************/
-/********************************************/
-
-std::shared_ptr<cadmium::dynamic::modeling::model> receiver1 = cadmium::dynamic::translate::make_dynamic_atomic_model<Receiver, TIME>("receiver1");
+    using
+        info=cadmium::logger::logger<cadmium::logger::logger_info,
+            cadmium::dynamic::logger::formatter<TIME>,
+                oss_sink_provider>;
+    using
+        debug=cadmium::logger::logger<cadmium::logger::logger_debug,
+            cadmium::dynamic::logger::formatter<TIME>,
+                oss_sink_provider>;
+    using
+        state=cadmium::logger::logger<cadmium::logger::logger_state,
+            cadmium::dynamic::logger::formatter<TIME>,
+                oss_sink_provider>;
+    using
+        log_messages=cadmium::logger::logger<cadmium::logger::logger_messages,
+            cadmium::dynamic::logger::formatter<TIME>,
+                oss_sink_provider>;
+    using
+        routing=cadmium::logger::logger<cadmium::logger::logger_message_routing,
+            cadmium::dynamic::logger::formatter<TIME>,
+                oss_sink_provider>;
+    using
+        global_time=cadmium::logger::logger<cadmium::logger::logger_global_time,
+            cadmium::dynamic::logger::formatter<TIME>,
+                oss_sink_provider>;
+    using
+        local_time=cadmium::logger::logger<cadmium::logger::logger_local_time,
+            cadmium::dynamic::logger::formatter<TIME>,
+                oss_sink_provider>;
+    using
+        log_all=cadmium::logger::multilogger <info,
+            debug, state, log_messages, routing, global_time, local_time>;
+
+    using logger_top=cadmium::logger::multilogger<log_messages, global_time>;
+
+
+    /*******************************************/
+
+
+
+    /********************************************/
+    /****** APPLICATION GENERATOR *******************/
+    /********************************************/
+    string input_data_control =
+        "../test/data/receiver/receiver_input_test.txt";
+    const char * i_input_data_control = input_data_control.c_str();
+
+    std::shared_ptr<cadmium::dynamic::modeling::model>
+        generator = cadmium::dynamic::translate::make_dynamic_atomic_model
+            <ApplicationGen, TIME,const char* >
```

Comment:
A space is required before const char* to maintain uniformity in parameters. 
(all the parameter have a single space after comma)

Reply:
Solved

Reply:
Thanks for correcting it and no space would be required after char*

Reply:
solved